docs/general-faq/game-url-files.md

```diff
@@ -0,0 +1,20 @@
+---
+icon: material/arrow-top-right-thick
+description: How to fix my shortcuts showing up as .url files or without an icon?
+---
+
+# :material-arrow-top-right-thick: How do I fix my shortcuts showing up as .url files or without an icon?
+
+**Simply set the shortcut to open in "Internet Browser".**
```

Comment:
Sometimes, people have already selected something else to open the URL file. Maybe change the steps into using the 'Open With' context menu to fix this?

Reply:
I thought about that, but I don't know if it would show up for a .url file.